lib/versions.js

```diff
@@ -86,9 +101,9 @@ function fetchLatestTag(repo) {
   );
 }
 
-async function getVersionFromToolVersionsFile() {
+async function getVersionFromToolVersionsFile(path) {
   try {
-    const toolVersions = await fs.readFile(".tool-versions", {
+    const toolVersions = await fs.readFile(path + ".tool-versions", {
```

Comment:
Please use `path.join` here. Primarily, it looks like this will crash if user provides `tool-versions: ./tmp` (no trailing slash). Also, don't forget about Windows :)

Reply:
should we use `path` module or implement some util function like
`const pathJoin = (parts, sep='/') => parts.join(sep).replace(new RegExp(sep+'{1,}', 'g'), sep);` ?

Reply:
`path` module is the way to go :) I see you did this so that's great!